src/ILVerify/tests/ILTests/SwitchTests.il

```diff
@@ -0,0 +1,260 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern System.Runtime
+{
+}
+
+.assembly SwitchTests
+{
+}
+
+.class public auto ansi beforefieldinit SwitchTestsType
+       extends [System.Runtime]System.Object
+{
+    .method static public hidebysig void Switch.Int32Value_Valid() cil managed
+    {
+        .maxstack 1
+        .locals init (
+            [0] int32
```

Comment:
These locals are unnecessary. The value can be loaded directly onto the stack right before the switch.